source/ncengine/ui/editor/impl/ComponentWidgets.cpp

```diff
@@ -803,13 +804,22 @@ void ParticleEmitterUIWidget(graphics::ParticleEmitter& emitter, EditorContext&,
     ui::PropertyWidget(particle_emitter_ext::scaleOverTimeFactoryProp, emitter, &ui::DragFloat, step, minFactor, maxFactor);
 }
 
+void DirectionalLightUIWidget(graphics::DirectionalLight& light, EditorContext&, const std::any&)
+{
+    ui::InputColor3(light.color, "color");
+}
+
 void PointLightUIWidget(graphics::PointLight& light, EditorContext&, const std::any&)
 {
     constexpr auto step = 0.1f;
     constexpr auto min = 0.0f;
     constexpr auto max = 1200.0f;
+#ifndef NC_USE_DILIGENT
```

Comment:
If we're only using one of the colors going forward, will you make an issue to remove it? I would, but I'm not sure what the plan is/if this is correct.

Reply:
Done, https://github.com/NcStudios/NcEngine/issues/807